src/server/lib/dbfile.ts

```diff
@@ -0,0 +1,154 @@
+import { readFile, writeFile, rename, readdir } from "fs/promises"
+import path from "node:path"
+
+const DATADIR = `./.data`
+const TICK = 500
+
+export type Write = ReturnType<typeof writeFile>
+
+// this is fucking stupid why did i write this
+
+class Activity {
+    
+    _write: () => Promise<any>
+    destroy: () => void
+
+    goal: number = Date.now()
+    lastWrt: number = Date.now()
```

Comment:
last huh? Did you mean `lastWrite`? Name it that dumbass
```suggestion
    lastWrite: number = Date.now()
```

Reply:
I'm gonna fucking kill myself